lib/validators/movies/list.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).optional(),
```

Comment:
I just wanted to stick with the idea that we decided to rename the column internally but didn't want to affect how clients interact with the API. Sound reasonable?